tests/integration/test_tickets.py

```diff
@@ -141,3 +142,10 @@ def test_tickets_DELETE(client, auth_headers):
     # NOT FOUND - Trying to delete a non-existing ticket or an already deleted ticket
     assert is_valid(response, 404)
     assert response.json == {'message': 'Ticket not found'}
+
+@pytest.mark.usefixtures("empty_test_db")
+class TestFixtures:
+    def test_create_ticket(self, create_ticket):
+        ticket = create_ticket()
+        assert ticket
+        assert ticket.issue == "Leaky pipe"
```

Comment:
This is probably better placed in the `tests/unit/test_ticket.py` file since we're not making a request through an endpoint and this more of a unit test in the first place. I do like how you broke it out into a separate class so that it is easy to see that this is clearly a different flavor of tests from the rest in the file and the class name tells us exactly what it is doing, testing fixtures. Let's get this merged and we can always move it later the next time if need be.